### DISCUSSION TITLE
Skewing an image

### DISCUSSION THREAD

**Discussion by evillurker** [ImageMagick/ImageMagick#3856] [Consulting]
Hello, I tried so many things so I turn to you guys no other choice..
I have a script that puts an image on another image, and I want to skew it to the right (so the upper right corner will be skewed a bit to the right)

right now im using this argument because that's how it was defaulted on the script that I downloaded:
+distort Perspective  "0,0 -30,4  0,200 -30,200  40,200 0,200  40,0 0,0"

But that doesn't do much. 
Any idea how can I set the right numbers for the image to be skewed?
image size is approximately 160x260

**Comment by fmw42** [comment]
Have yo tried -shear. See https://imagemagick.org/script/command-line-options.php#shear

**Comment by snibgo** [comment]
I'm not sure what you mean by "skew it to the right (so the upper right corner will be skewed a bit to the right)".

What size is the input? What size should the output be? Should the output contain all of the input, or do you expect some of the input image to be trimmed off?

Perhaps you can show an example.

And, the usual questions: what version of IM, on what platform?
